src/uproot/writing/_cascade.py

```diff
@@ -2003,7 +1996,7 @@ def __init__(
         self._begin = 100
 
     def __repr__(self):
-        return "{0}({1}, {2}, {3}, {4}, {5}, {6}, {7}, {8}, {9})".format(
+        return "{}({}, {}, {}, {}, {}, {}, {}, {}, {})".format(
```

Comment:
This one is easier to read in this form vs. an f-string.

Reply:
I agree.